frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -169,6 +169,20 @@ function ReaderHighlight:init()
         }
     end)
 
+    -- User hyphenation dict
+    self:addToHighlightDialog("11_user_dict", function(_self)
+        return {
+            text= _("Hyphenation"),
+            show_in_highlight_dialog_func = function()
+                return _self.ui.userhyph:_checked() and not _self.selected_text.text:find("[ ,;-%.\n]")
```

Comment:
If `:_checked()` is public (as it is here, used by this other module), don't use a underscore, use just `:checked()`. Or give it a better name: `:isEnabled()` or `:isUsable()` or `:isAvailable()`